### PR TITLE
WM-13-cleaner

### PR DESCRIPTION
**Pull request by europaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
+| UPPER_THRESHOLD                | float  | Any data points exceeding this threshold should be considered outliers.                                          |
+| LOWER_THRESHOLD                | float  | Any data points below this threshold should be considered outliers.                                              |
+| RATE_OF_CHANGE_UPPER_THRESHOLD | float  | Any data point where the rate of change (per second) is greater than this value should be considered an outlier. |
+| RATE_OF_CHANGE_LOWER_THRESHOLD | float  | Any data point where the rate of change (per second) is less than this value should be considered an outlier.    |
+| OUTLIER_POLICY                 | string | What to do with outliers: keep, remove, smooth                                                                   |
+| INPUT_DATA_LABEL               | string | The input label on which to detect outliers.                                                                     |
+| INPUT_TIME_LABEL               | string | JSON key for the timestamp. The timestamp should in epoch format (int).                                          |
```
Comment (christosWeeve) [inline README.md:41]: Is this a mandatory value? And if yes, do we have a module to add it in this format so this module can get the right input?

**Review by christosWeeve — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (europaul) [reply README.md:41]: yes, I was thinking the [timestamp](https://github.com/weeve-modules/timestamp) module

**Review by christosWeeve — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,54 +1,57 @@
...
 
-|                |                                       |
-| -------------- | ------------------------------------- |
-| Name           | Cleaner                               |
-| Version        | v1.0.0                                |
+|                |                                                                       |
+| -------------- | --------------------------------------------------------------------- |
+| Name           | Outlier Detector                                                      |
+| Version        | v2.0.0                                                                |
 | Dockerhub Link | [weevenetwork/cleaner](https://hub.docker.com/r/weevenetwork/cleaner) |
```
Comment (kubagrzelak) [inline README.md]:
```suggestion
| Dockerhub Link | [weevenetwork/outlier-detector](https://hub.docker.com/r/weevenetwork/outlier-detector) |
```

Comment (kubagrzelak) [reply README.md]: We should have a new DockerHub repo for that. Probably I will delete the cleaner repo later.

File: makefile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 SHELL := /bin/bash # to enable source command in run_app
 
 MODULE=weevenetwork/cleaner
```
Comment (kubagrzelak) [inline makefile]:
```suggestion
MODULE=weevenetwork/outlier-detector
```

**Review by kubagrzelak — CHANGES_REQUESTED** [review]
You forgot to change module name to `outlier-detector` in other parts of the codebase including docker-compose etc.

**Pull request merged**